eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -49,26 +49,34 @@ jobs:
     # Linux x64
     - ${{ if eq(parameters.platform, 'Linux_x64') }}:
       - ${{ if eq(parameters.jobParameters.interpreter, '') }}:
-        - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
+        - ${{ if and(eq(parameters.jobParameters.testScope, 'outerloop'), eq(parameters.jobParameters.runtimeFlavor, 'mono')) }}:
           - Centos.7.Amd64.Open
           - RedHat.7.Amd64.Open
-          - Debian.9.Amd64.Open
-          - Ubuntu.1604.Amd64.Open
-          - Ubuntu.1804.Amd64.Open
-          - SLES.12.Amd64.Open
           - SLES.15.Amd64.Open
-          - (Fedora.30.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30-helix-20200512010621-4f8cef7
           - (Fedora.32.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-32-helix-20200512010618-efb9f14
           - (Ubuntu.1910.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-19.10-helix-amd64-cfcfd50-20191030180623
           - (Debian.10.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64-bfcd90a-20200121150006
-        - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
-          - Centos.7.Amd64.Open
-          - RedHat.7.Amd64.Open
-          - (Debian.10.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64-bfcd90a-20200121150006
-          - Ubuntu.1604.Amd64.Open
-          - Ubuntu.1804.Amd64.Open
-          - SLES.15.Amd64.Open
-          - (Fedora.30.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30-helix-20200512010621-4f8cef7
+        - ${{ if or(ne(parameters.jobParameters.testScope, 'outerloop'), ne(parameters.jobParameters.runtimeFlavor, 'mono')) }}:
```

Comment:
do I parse this yml right that we have a different set of queues for mono vs. coreclr outerloop runs? and that there is no difference between isFullMatrix true/false for mono?

Reply:
Yeah I was trying to have mono run on the latest OSs to balance payload and see what people thought about it. If there are any concerns with this I could of course change it.